ci/run.sh

```diff
@@ -8,8 +8,17 @@ if [ "$1" = "aarch64-apple-darwin" ] ; then
 	export CARGO_TARGET_AARCH64_APPLE_DARWIN_RUNNER=echo
 fi
 
-cargo test --manifest-path testcrate/Cargo.toml --target $1 -vv
-cargo test --manifest-path testcrate/Cargo.toml --target $1 -vv --release
+# Use cargo package to ensure we don't rely on any files excluded by Cargo.toml
+cargo package --allow-dirty
```

Comment:
Could this be split out to a separate CI step for one platform to test the packaged version? I'd prefer to still do at least some testing of the in-tree version on CI.

Reply:
@alexcrichton I had a look, but I'm not familiar enough with GitHub workflows to make this happen :grimacing: .

As a halfway house, I've moved the logic to be within the `if [ "$1" = "x86_64-unknown-linux-gnu" ]` block, which has the same effect but without it being a separate CI step. What do you think?